packages/compass-schema/src/components/type/type.jsx

```diff
@@ -151,9 +149,8 @@ class Type extends Component {
         <div className="schema-field-type" />
         {subtypes}
```

Comment:
Definitely a lot of room for improvement here agreed, I think as a drive by fix we're already doing improvement. Down the line it will be nice to spend the time to make these components nice.